### PR TITLE
Stele Road - scrnshot edit

### PR DESCRIPTION
**Pull request by vtange**

### CODE REVIEW

File: QUEST_LV_0100.tsv [hunk 1]
```diff
@@ -812,18 +812,18 @@ QUEST_LV_0100_20150317_000811	Who cherished such traitor? Don't you think there
...
-QUEST_LV_0100_20150317_000815	Thanks.{nl}Achat must have drawn to the power of the magic that is casted upon the monument.
+QUEST_LV_0100_20150317_000815	$Thanks.{nl}Achat must have been drawn to the magic power cast upon the monument.
 QUEST_LV_0100_20150317_000816	You look live a wood so it's curious to find out that you are interested in stone plates.
 QUEST_LV_0100_20150317_000817	I am sorry, but we can't find the one that we needed among the fragments you brought.{nl}I hope it's not already eaten.
-QUEST_LV_0100_20150317_000818	Don't you think we would be able to find the fragments that Treeambulo swallowed if we burn it?{nl}I have the woods that I stacked before so we push it into it while the woods are burning.
-QUEST_LV_0100_20150317_000819	Be careful.{nl}It will be dangerous if Treeambulo in flame runs into the camp.
+QUEST_LV_0100_20150317_000818	$Don't you think we would be able to find the stone pieces that a Tree Ambulo swallowed if we burn it?{nl}I have stacked woods there, so we can push it in when the wood is burning.
```
Comment (terrapin47) [inline QUEST_LV_0100.tsv]: "I have stacked wood there"

File: QUEST_LV_0100.tsv [hunk 2]
```diff
@@ -1338,11 +1338,11 @@ QUEST_LV_0100_20150317_001337	{memo X}
 QUEST_LV_0100_20150317_001338	Are you going to defeat Netherbovines? {nl}Splendid. I am counting on you.
 QUEST_LV_0100_20150317_001339	The magic to control time is the essence of all magic.{nl}Agailla Flurry told me.
 QUEST_LV_0100_20150317_001340	{memo X}
-QUEST_LV_0100_20150317_001341	I heard Nederbovines are going nuts outside Fedimian.{nl}As a person undergoing training, I am ashamed that I can't go out and fight.
```
Comment (terrapin47) [inline QUEST_LV_0100.tsv:1341]: "Netherbovine" is a boss ( singular ).

**Comment by terrapin47** [comment]
Looks good to me, after resolving the comments :dancer:

File: QUEST_LV_0100.tsv [hunk 3]
```diff
@@ -812,18 +812,18 @@ QUEST_LV_0100_20150317_000811	Who cherished such traitor? Don't you think there
...
 QUEST_LV_0100_20150317_000821	I am a treasure hunter and not a monster hunter.{nl}I don't want to add my gravestone here by touching it.
-QUEST_LV_0100_20150317_000822	ve Collected all the debris out of the deal is over Raymond. {nl} brought back the parcel to Raymond.
-QUEST_LV_0100_20150317_000823	Ethan told you to give it to me?{nl}Hmm. It will take sometime to restore it.{nl}
-QUEST_LV_0100_20150317_000824	Can you take the restored gravestone fragments to my assistant at Irody Square?{nl}I think something good will happen when we finish restoring it.
+QUEST_LV_0100_20150317_000822	$I've collected all the debris Raymond requested. {nl} Bring them back to Raymond.
+QUEST_LV_0100_20150317_000823	$Ethan told you to give this to me?{nl}Hmm. It will take sometime to restore it.{nl}
+QUEST_LV_0100_20150317_000824	$Can you take these restored gravestone fragments to my assistant at Irody Square?{nl}I think something good will happen when we finish restoring it.
 QUEST_LV_0100_20150317_000825	Epigraphist Smid
-QUEST_LV_0100_20150317_000826	This.. Our master did it!{nl}Give it to me.
+QUEST_LV_0100_20150317_000826	$This.. Our master did it!{nl}Can I have it?
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: This.. -> This...
"Can I have it?" has a very different tone compared to "Give it to me", which is what the original says. The original says: "Give it to me, please."

File: QUEST_LV_0100.tsv [hunk 4]
```diff
@@ -1320,12 +1320,12 @@ QUEST_LV_0100_20150317_001319	A person who uses a demon's name.. He must be craz
...
-QUEST_LV_0100_20150317_001326	It's okay even if you are not a swordsman.{nl}It's okay even if you are not selected.{nl}
-QUEST_LV_0100_20150317_001327	The light that Ruklys will leave to you should be used for this chaotic world.{nl}If you only concentrate on killing, you may also be killed.{nl}
-QUEST_LV_0100_20150317_001328	I am looking for a person who will help me with my research.{nl}All the support for the research has stopped since the research is about the traitor Ruklys.
+QUEST_LV_0100_20150317_001325	$The King tyrannized more as his spirit waned, and devastated his citizens' lives.{nl}I didn't die with him, so I am going to leave his spirit on this monument.
+QUEST_LV_0100_20150317_001326	$It's okay even if you are not a swordsman.{nl}It's okay even if you are not the chosen.{nl}
+QUEST_LV_0100_20150317_001327	$The light that Ruklys will leave you with should be used against this chaotic world.{nl}If you only concentrate on killing, you may also be killed.{nl}
+QUEST_LV_0100_20150317_001328	$I am looking for a person who will help me with my research.{nl}I lost all support when they found out the research is about the traitor Ruklys.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: is -> was?

File: QUEST_LV_0100.tsv [hunk 5]
```diff
@@ -1338,11 +1338,11 @@ QUEST_LV_0100_20150317_001337	{memo X}
...
+QUEST_LV_0100_20150317_001341	I heard Netherboviness are going nuts outside Fedimian.{nl}As a person undergoing training, I am ashamed that I can't go out and fight.
 QUEST_LV_0100_20150317_001342	Don't say poison is unfair.{nl}You gotta do anything to survive. You can't live out this world when you just want to be fair.
-QUEST_LV_0100_20150317_001343	Are you going out to defeat Nederbovines outside Fedimian?{nl}If it weren't you, I would have fought.{nl}Good luck.
+QUEST_LV_0100_20150317_001343	Are you going out to defeat Netherboviness outside Fedimian?{nl}If it weren't you, I would have fought.{nl}Good luck.
 QUEST_LV_0100_20150317_001344	Even after this beautiful city got destroyed, there are still many threats.{nl}I can go out and fight, but.. I won't.{nl}There are many people out there who will fight without any rewards.{nl}
-QUEST_LV_0100_20150317_001345	Nederbovines.. It is fortunate that we have some to fight against them.
+QUEST_LV_0100_20150317_001345	Netherboviness.. It is fortunate that we have some to fight against them.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: extra period

File: QUEST_LV_0100.tsv [hunk 6]
```diff
@@ -1891,10 +1891,10 @@ QUEST_LV_0100_20150317_001890	Please go to the Mausoleum.{nl}We should find out
...
-QUEST_LV_0100_20150317_001895	I am trying to find one of them by using small trees at Balkonas wilderness.{nl}With a wooden piece from the monster, we will make an exploration stick to find it.
-QUEST_LV_0100_20150317_001896	There's a piece below Secrus Bridge that is suspected to be a piece from the monument.{nl}But Achat is around there so I can't reach near.
-QUEST_LV_0100_20150317_001897	TreeAmbulo that lives in Gaisra Hall likes to collect stone pieces.{nl}Maybe, the pieces from the monument could be there.
+QUEST_LV_0100_20150317_001894	$Ruklys committed treason, so his monument saw prejudice and neglect.{nlIt was broken into five pieces, which all disappeared.{nl}
+QUEST_LV_0100_20150317_001895	$I am trying to find one of them via the Short Trees at Balkonas Wilderness.{nl}With a wooden piece from that monster, we can make an exploration stick to find it with.
+QUEST_LV_0100_20150317_001896	$There's a piece below Sekrus Bridge that is suspected to be a piece from the monument.{nl}But Achat is around there so I can't reach near.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Sekrus -> Seklus

File: QUEST_LV_0100.tsv [hunk 7]
```diff
@@ -5300,22 +5300,22 @@ QUEST_LV_0100_20150317_005299	Defeat Rexipher's Subordinates
...
+QUEST_LV_0100_20150317_005308	$Made the Crude Explorer stick using the Short Tree Branches. Use it to find the First Piece of Gravestone.
 QUEST_LV_0100_20150317_005309	Found the First Piece of Gravestone. Go back to Raymond. 
 QUEST_LV_0100_20150317_005310	Find the Second Piece of Gravestone under Sekrus Bridge
 QUEST_LV_0100_20150317_005311	Raymond told you where the Second Piece of Gravestone might be. First, make sure this unique slate is actually the Second Piece of Gravestone.
 QUEST_LV_0100_20150317_005312	The unique slate wasn't the real thing, but you defeated Echat and got the Second Piece of Gravestone. Go back to Raymond. 
 QUEST_LV_0100_20150317_005313	Defeat Echat
 QUEST_LV_0100_20150317_005314	Collect the Dusty Ruins
-QUEST_LV_0100_20150317_005315	Raymond asked you to collect pieces of the Dusty Ruins from Gaisra Battlefield. He told you to defeat the Treeambuloes and get the Dusty Ruins back if they happen to steal any from you. 
+QUEST_LV_0100_20150317_005315	$Raymond asked you to collect pieces of the Dusty Ruins from Gaisra Hall. He told you to defeat the Tree Ambuloes and get the Dusty Ruins back if they happen to steal any from you.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Gaisra Hall -> Gaisra Hall Site

File: QUEST_LV_0100.tsv [hunk 8]
```diff
@@ -5300,22 +5300,22 @@ QUEST_LV_0100_20150317_005299	Defeat Rexipher's Subordinates
...
 QUEST_LV_0100_20150317_005313	Defeat Echat
 QUEST_LV_0100_20150317_005314	Collect the Dusty Ruins
-QUEST_LV_0100_20150317_005315	Raymond asked you to collect pieces of the Dusty Ruins from Gaisra Battlefield. He told you to defeat the Treeambuloes and get the Dusty Ruins back if they happen to steal any from you. 
+QUEST_LV_0100_20150317_005315	$Raymond asked you to collect pieces of the Dusty Ruins from Gaisra Hall. He told you to defeat the Tree Ambuloes and get the Dusty Ruins back if they happen to steal any from you. 
 QUEST_LV_0100_20150317_005316	Gathered enough Dusty Ruins. Give them to Raymond. 
-QUEST_LV_0100_20150317_005317	Burn Treeambuloes and get Carbonized Pieces
-QUEST_LV_0100_20150317_005318	Raymond told you to light the firewood in Gaisra Battlefield and defeat the Treeambuloes, then burn them to get carbonized pieces. 
+QUEST_LV_0100_20150317_005317	$Burn Tree Ambuloes and get Carbonized Pieces
+QUEST_LV_0100_20150317_005318	$Raymond told you to light the firewood in Gaisra Hall and defeat the Tree Ambuloes, then burn them to get carbonized pieces.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Gaisra Hall -> Gaisra Hall Site

File: QUEST_LV_0100.tsv [hunk 9]
```diff
@@ -5386,7 +5386,7 @@ QUEST_LV_0100_20150317_005385	{memo X}
 QUEST_LV_0100_20150317_005386	{memo X}Restore Ruklys Memorial Stone
 QUEST_LV_0100_20150317_005387	Restron Ruklys memorial stone and get Ruklys legacy
 QUEST_LV_0100_20150317_005388	Help epigraphis Smid
-QUEST_LV_0100_20150317_005389	Magburk attacked and killed Smid as he was trying to get the rubbings of Ruklys memorial stone. Defeat the Marbuk for Smid's revenge.
+QUEST_LV_0100_20150317_005389	$Magburk emerged and killed Smid as he was trying to get the rubbings of Ruklys memorial stone. Defeat Marburk for to avenge Smid.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Defeat Marburk for to avenge Smid. -> Defeat Marburk to avenge Smid.

File: QUEST_LV_0100.tsv [hunk 10]
```diff
@@ -6038,9 +6038,9 @@ QUEST_LV_0100_20150317_006037	Find the Fourth Piece of Gravestone
 QUEST_LV_0100_20150317_006038	{memo X}Treasure Hunter says he's sure that Tama swallowed the relics. Defeat Tama and get the relics. 
 QUEST_LV_0100_20150317_006039	Tama has the fourth gravestone piece as the treasure hunter said. Go and give it to Ethan.
 QUEST_LV_0100_20150317_006040	Defeat Tama and get %s
-QUEST_LV_0100_20150317_006041	Collect sap of Small Tree
-QUEST_LV_0100_20150317_006042	Treasure Hunter Ethan asked you to collect the sap of Small Tree for its worth. Attack the sap collector to the small tree and get the sap after some time.
-QUEST_LV_0100_20150317_006043	Collected enough Small tree sap. Go to Ethan for rewards. 
+QUEST_LV_0100_20150317_006041	$Collect sap of Short Tree
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv:6041]: to get sap. ->  to get their sap?

File: ETC.tsv [hunk 11]
```diff
@@ -5260,8 +5260,8 @@ ETC_20150317_005259	{memo X}Monsters have appeared after you collected the Bisho
 ETC_20150317_005260	Bishop's Tomb
 ETC_20150317_005261	{memo X}It doesn't seem like the slate you are looking for.
 ETC_20150317_005262	Achat appeared just like Raymond said!
-ETC_20150317_005263	{memo X}Smid was murdered!{nl}Please defeat Magbirk!
-ETC_20150317_005264	Magbirk killed the Epigraphist Smid!{nl}Please defeat Magbirk!
+ETC_20150317_005263	{memo X}Smid was murdered!{nl}Please defeat Magburk!
+ETC_20150317_005264	$Magburk killed the Epigraphist Smid!{nl}Please defeat Magburk!
```
Comment (Sunaries) [inline ETC.tsv]: Magburk killed the Epigraphist Smid -> Magburk killed Epigraphist Smid? 
Please defeat -> Defeat

File: QUEST_LV_0100.tsv (hunk 10 above)
Comment (vtange) [inline QUEST_LV_0100.tsv:6041]: fixed it. this highlight was off by one line.

**Comment by imcgames** [comment]
Thank you :smile:

**Pull request merged**